include/mbedtls/md.h

```diff
@@ -334,9 +338,9 @@ int mbedtls_md( const mbedtls_md_info_t *md_info, const unsigned char *input, si
  * \param path     The input file name.
  * \param output   The generic message-digest checksum result.
  *
- * \return         \c 0 on success,
- *                 #MBEDTLS_ERR_MD_FILE_IO_ERROR if file input failed, or
- *                 #MBEDTLS_ERR_MD_BAD_INPUT_DATA if \p md_info was NULL.
+ * \return         \c 0 on success.
+ * \returns        #MBEDTLS_ERR_MD_FILE_IO_ERROR if file input failed.
```

Comment:
Suggestion: Improve the pre-existing wording, e.g. `#MBEDTLS_ERR_MD_FILE_IO_ERROR on an I/O error accessing the file pointed to by \p path.`

Reply:
fixed